<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblOriginal">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>36</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessed">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>36</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>4</y>
      <width>113</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Window</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>8</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="labl_mask">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>304</y>
      <width>309</width>
      <height>209</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCamera_0"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="imageProcessingMenu">
    <property name="title">
     <string>Image Processing</string>
    </property>
    <addaction name="actionImageProcessingSettings"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="imageProcessingMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCamera_0">
   <property name="text">
    <string>Camera 0</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImageProcessingSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
